apps/siiges-app/pages/solicitudes/nuevaSolicitud/index.jsx

```diff
@@ -26,135 +15,25 @@ const steps = [
-                xs={9}
-                justifyContent="end"
-                alignItems="center"
-                sx={{ textAlign: 'left' }}
-              >
-                <Box
-                  alignItems="center"
-                  display="flex"
-                  sx={{
-                    backgroundColor: 'darkGray',
-                    width: 230,
-                    ml: 50,
-                    py: 1,
-                    px: 1,
-                    borderRadius: 20,
-                  }}
-                >
-                  <LoadCircle state="11" />
-                  <Typography
-                    alignItems="center"
-                    variant="p"
-                    sx={{
-                      color: 'white',
-                      ml: 3,
-                    }}
-                  >
-                    1 de 9 completado
-                  </Typography>
-                </Box>
-              </Grid>
-              <Grid
-                item
-                xs={3}
-                sx={{ textAlign: 'right', alignItems: 'end', mt: 1 }}
-              >
-                <ButtonStyled
-                  text="Terminar"
-                  alt="Terminar solicitud"
-                  type="success"
-                />
-                <span>&nbsp;&nbsp;</span>
-                <ButtonStyled
-                  text={<ArrowForwardIosIcon sx={{ height: 14 }} />}
-                  alt={<ArrowForwardIosIcon sx={{ height: 14 }} />}
-                  type="success"
-                />
-              </Grid>
-            </Grid>
-            <Grid container sx={{ mt: 3 }}>
-              <Grid item xs={12}>
-                <Typography variant="h6">Datos de institucion</Typography>
-              </Grid>
-              <Grid item sx={{ ml: 15, width: '100%' }}>
-                <InstitucionData />
-              </Grid>
-              <Grid
-                item
-                xs={12}
-                sx={{ textAlign: 'right', alignItems: 'end', mt: 1 }}
-              >
-                <ButtonStyled
-                  text="Terminar"
-                  alt="Terminar solicitud"
-                  type="success"
-                />
-                <span>&nbsp;&nbsp;</span>
-                <ButtonStyled
-                  text={<ArrowForwardIosIcon sx={{ height: 14 }} />}
-                  alt={<ArrowForwardIosIcon sx={{ height: 14 }} />}
-                  type="success"
-                />
-              </Grid>
-            </Grid>
-          </CardContent>
-        </Card>
-      </Container>
-    </ThemeProvider>
+    <Layout type={false}>
+      <ModuleHeader
+        steps={steps}
+        type="Nueva solicitud"
+        date="22 de Agosto 2022"
+        nextModule={nextModule}
+        module={module}
+      />
+      {module === 0 && <DatosGenerales />}
+      {module === 1 && <Plantel />}
+      {module === 2 && <PlanEstudios />}
+      {module === 3 && <Anexos />}
+      {module === 4 && <EvaluacionCurricular />}
```

Comment:
In this part I'd prefer it to be more explicit, I can't get what are you trying to do